assets/json/menu.json

```diff
@@ -532,16 +544,36 @@
           "layout": {
             "x_anchor": "center",
             "y_anchor": "middle",
-            "x_offset": 0.25,
-            "y_offset": -0.05
+            "x_offset": 0.15,
+            "y_offset": -0.1
           }
         },
-        "gamelobbyplayerlabel5": {
-          "comment": "The label at the bottom-right for opponent player's info",
+        "volumeinput": {
+          "comment": "The volume slider in Settings.",
+          "type": "Slider",
+          "data": {
+            "bounds": [40, 0, 300, 0],
+            "range": [1, 100],
+            "value": 50,
+            "tick": 10,
+            "snap": true,
```

Comment:
See my other comment for explanation, but if we could make this have range [0, 1] with default value of 0.5 that would be great. Also, I feel like the snapping isn't really needed for this slider (and I guess the tick as well).